Components/JsonFileLoader/JsonFileLoaderProducer.php

```diff
@@ -18,6 +18,9 @@ class JsonFileLoaderProducer extends Producer
 {
     const OPTION_FILENAME = 'filename';
     const OPTION_BASE_PATH = 'base_path';
+    const OPTION_TYPE = 'type';
```

Comment:
We don't need to have this constant on the producer as they are defined in the protocol.